commons/zenoh-keyexpr/src/key_expr/utils.rs

```diff
@@ -228,12 +205,15 @@ impl<const N: usize> Split<[u8; N]> for [u8] {
     }
 }
 
+#[allow(dead_code)]
```

Comment:
If this is unused shouldn't we just remove it ?

Reply:
That `Utf` trait is used by the `utfdbg` macro. And said macro is kept because it's "useful when introducing new matchers to debug them".